contracts/rbn-staking/GaugeController.vy

```diff
@@ -551,6 +556,23 @@ def vote_for_gauge_weights(_gauge_addr: address, _user_weight: uint256):
 
     log VoteForGauge(block.timestamp, msg.sender, _gauge_addr, _user_weight)
 
+@external
```

Comment:
this is the method that distributes to all of the gysr staking contracts based on weight voted on. gauge controller already takes care of voting logic based on sRBN (veCRV) balance